Objects/bytearrayobject.c

```diff
@@ -224,7 +224,7 @@ PyByteArray_Resize(PyObject *self, Py_ssize_t requested_size)
             return -1;
         }
         memcpy(sval, PyByteArray_AS_STRING(self),
-               Py_MIN(requested_size, Py_SIZE(self)));
+               Py_MIN((size_t)requested_size, (size_t)Py_SIZE(self)));
```

Comment:
You can use `size` instead of `(size_t)requested_size`.

Reply:
Yes, but I'd prefer to use `const size_t size = (size_t) requested_size;` then. It would be one more change.